metadata-ingestion/setup.py

```diff
@@ -59,6 +59,7 @@ def get_long_description():
     "jsonref",
     "jsonschema",
     "ruamel.yaml",
+    "croniter==1.4.1",
```

Comment:
Can we avoid adding an extra dependency to the core framework?

We're only using this for validation?

Reply:
Yes, it is only for validation. Since I’m  pushing an mcp I wanted some level of validation.
Suggestions are welcome

Reply:
Updated, please take another look